README.md

```diff
@@ -45,6 +45,8 @@ To determine where to install a dependency:
 
 ## Building
 
+### Amd64
+
 Build for all platforms:
 
 - `yarn build:all`
```

Comment:
Could we have build:all build arm64 as well, so that it's not two separate steps?

Reply:
That would be possible at least for those builds that could directly be built on an amd64 machine – for those it seems to be enough to have the proper env variable(s) set in the `build.mjs`. If a Debian package wouldn't be part of a regular release cycle anyway and with the snap releases still probably requiring a VM (haven't tested the feasibility of that on macOS yet), that would mean just adding the arm64 AppImage build to it.

I'd suggest I'd modify the `package.json` accordingly to, for now, include the arm64 AppImage build with `build:all`, and rewrite the README so that we'd have that as part of the "generic build section", while keeping the "more involved" arm64 builds/steps as a separate section for anyone who'd dare to give building on native arm64 machines a try?

Reply:
I agree with this approach 🙂 I'd add the builds which can be built from MacOS to the `build:all` command (Debian could be included too, correct?), and leave those who require a more involved setup in a separate command